### ISSUE TITLE
Code is inserted at the top of file, not at cursor

### ISSUE THREAD

**Issue by TonyGravagno** [coozieki/php-implementor#3]
I'm hoping that this extension will insert the interface implementation at the current cursor position, when invoked from the right-click context menu. It chooses to search from the top of the file for some condition, and inserts the code where it thinks it belongs. Obviously where it's inserting is incorrect. I'm guessing your code doesn't recognize multi-line `use` blocks.

Example from top of code file:

```php

/** @package PROJECT */

namespace MAIN\Framework;

use MAINU\Enums\State;
use MAIN\Framework\BaseController;
use MAIN\Modules\ModuleManager;

use MAIN\Hooks\{
        // <<<<<<<<< functions inserted here
	IHandleHooks,
	InitHooks,
};
use MAIN\Init\{
	Activate,
	Deactivate,
};
```

Thanks.

**Comment by coozieki** [comment]
@TonyGravagno yes, i didn't make a use blocks recognition as i didn't really use it because of IDE's auto imports :) I'll add it soon.

Code insertion at cursor is a good idea, i'll see what i can do about it.

Thanks for your feedback!
